### ISSUE TITLE
Outdated info

### ISSUE THREAD

**Issue by ErikGranse** [MicrosoftDocs/appcenter-docs#521]
I'm looking at the API (https://openapi.appcenter.ms/#/crash/crashGroups_list) and nearly every API call listed on this page has been deprecated or removed. No idea how to accomplish crash report downloading.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: e8b051fd-562b-0f49-4f14-5c8414d5816d
* Version Independent ID: 12bb6146-07df-1801-869d-ff2b3a747fe8
* Content: [Exporting Crash Data - Visual Studio App Center](https://docs.microsoft.com/en-us/appcenter/gdpr/crashes-export#feedback)
* Content Source: [docs/gdpr/crashes-export.md](https://github.com/MicrosoftDocs/appcenter-docs/blob/live/docs/gdpr/crashes-export.md)
* Service: **vs-appcenter**
* GitHub Login: @iageoghe
* Microsoft Alias: **iageoghe**

**Comment by jwargo** [comment]
@ErikGranse I'm not sure what's up with that, I'll have someone from the team respond to you.

**Comment by ErikGranse** [comment]
Thanks. I eventually figured it out to an extent; according to https://docs.microsoft.com/en-us/appcenter/diagnostics/using-the-diagnostics-api, the API calls on this page are still available for UWP apps, but have been officially deprecated. I'm trying to track an Android/Xamarin app, so I have to use new the new API. The page I linked above has mapping from the deprecated to the new APIs, but I don't think they're functionally 1-to-1.

(Sorry for my tone in my original comment. Venting frustration to the aether.)
